### PR TITLE
Improve FAQ answer.

### PR DESCRIPTION
**Pull request by ssk2**

### CODE REVIEW

**Comment by ssk2** [comment]
retest this please

**Comment by benclarkwood** [comment]
retest this please

File: docs/pages/faq.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ title: Frequently Asked Questions
 
 #### __What is a DC/OS service?__
 
-  A [DC/OS service](https://dcos.io/docs/latest/overview/concepts/#dcos-service) is an instance of a service in a DC/OS cluster.
+  A [DC/OS service](https://dcos.io/docs/latest/overview/concepts/#dcos-service) is an instance of an existing service (typically a stateful, distributed service) in a DC/OS cluster. DC/OS services built using the DC/OS SDK implement schedulers which allow for orchestrated installation, update and backup/recovery.
```
Comment (sascala) [inline docs/pages/faq.md]: By "existing," you mean something like Apache Spark, which has an independent existence outside of the DC/OS context? What about people creating SDK services? Do they exist independently of DC/OS?

Suggest: "DC/OS services built using the DC/OS SDK implement schedulers, which all for orchestrated installation, update, and backup/recovery."

**Comment by ssk2** [comment]
@sascala, updated! Let me know what you think.

**Comment by sascala** [comment]
@ssk2 That makes so much more sense! 👍

**Pull request merged**